lib/App/Sqitch/Plan/Change.pm

```diff
@@ -221,6 +221,11 @@ sub verify_file {
     $self->verify_dir->file( $self->path_segments );
 }
 
+sub file {
+    my $self = shift;
+    return $self->is_deploy ? $self->deploy_file : $self->revert_file;
+}
+
```

Comment:
I don't think this is necessary; the `deploy_file` is always deployed, even if the action is a revert. The `revert_file` is used to revert the change — even if the change is a revert!

That said, the specification of revert changes, while supported by the plan file syntax from near the beginning, has never been implemented, and I'm don't recall ever working out how it should work, really. So I think you're best off using `deploy_file` exclusively.

Reply:
Updated.